core/consensus/common/src/block_import.rs

```diff
@@ -1,39 +1,16 @@
 
-use primitives::AuthorityId;
-use runtime_primitives::traits::{Block as BlockT, DigestItemFor};
+use primitives::{Blake2Hasher, AuthorityId, H256};
+use runtime_primitives::traits::{Block as BlockT, DigestItemFor, Header, As};
 use runtime_primitives::Justification;
+use runtime_primitives::generic::BlockId;
 
-/// Block import result.
-#[derive(Debug)]
-pub enum ImportResult {
-	/// Added to the import queue.
-	Queued,
-	/// Already in the import queue.
-	AlreadyQueued,
-	/// Already in the blockchain.
-	AlreadyInChain,
-	/// Block or parent is known to be bad.
-	KnownBad,
-	/// Block parent is not in the chain.
-	UnknownParent,
-}
+use client::backend;
+use client::client::{BlockOrigin, ImportResult, PrePostHeader, Client};
+use client::blockchain::{self, HeaderBackend};
+use client::call_executor::CallExecutor;
```

Comment:
```suggestion
use client::CallExecutor;
```
Do the same for line 8.

Reply:
(and merge them)

Reply:
These fixes should be done :) 👍